components/webflow/actions/create-collection-item/create-collection-item.mjs

```diff
@@ -1,53 +1,59 @@
-// legacy_hash_id: a_q1ioGv
-import { axios } from "@pipedream/platform";
+import webflow from "../../webflow.app.mjs";
 
 export default {
   key: "webflow-create-collection-item",
-  name: "Create Item",
+  name: "Create Collection Item",
   description: "Create new collection item",
-  version: "0.1.1",
+  version: "0.2.1",
   type: "action",
   props: {
-    webflow: {
-      type: "app",
-      app: "webflow",
+    webflow,
+    siteId: {
+      propDefinition: [
+        webflow,
+        "sites",
+      ],
+      optional: true,
     },
-    collection_id: {
-      type: "string",
+    collectionId: {
+      propDefinition: [
+        webflow,
+        "collections",
+        (c) => ({
+          siteId: c.siteId,
+        }),
+      ],
+    },
+    live: {
+      label: "Live",
+      description: "Indicate if the item should be published to the live site",
+      type: "boolean",
+      default: false,
     },
     name: {
+      label: "Name",
+      description: "Name given to the Item.",
       type: "string",
     },
     slug: {
+      label: "Slug",
+      description: "URL structure of the Item in your site.",
       type: "string",
     },
-    _archived: {
-      type: "boolean",
-    },
-    _draft: {
-      type: "boolean",
-    },
   },
-  async run({ $ }) {
+  async run() {
```

Comment:
We try to always add the `$` object in order to export a summary string to let the user know about the response like in this example
```javascript
  async run({ $ }) {
    const webflow = this.webflow._createApiClient();

    const response = await webflow.createItem({
      collectionId: this.collectionId,
      fields: {
        name: this.name,
        slug: this.slug,
        _archived: false,
        _draft: false,
      },
    }, {
      live: this.live,
    });

    $.export("$summary", `Successfully created item ${response._id}.`);

    return response;
  },
```
the same goes to the rest of components in this app

Reply:
Is returning the API Client method directly now, it's ok?